content/en/docs/reference/ios.md

```diff
@@ -19,14 +19,23 @@ The platform authenticators in iOS 16+ and iPadOS 16+ have the following capabil
 
 - creating and using passkeys that are backed up to iCloud Keychain
 - creating and using passkeys on/from another device, such as:
-  - an iPhone or iPad signed in to a different iCloud account
-  - an Android phone or tablet
+  - an iPhone or iPad signed in to a different iCloud account, using FIDO [Cross-Device Authentication](../terms#cross-device-authentication-cda)
```

Comment:
Is the full title of hybrid auth really, "FIDO Cross-Device Authentication"? If so, then we'll need to update the glossary to include "FIDO" because right now it's only "Cross-Device Authentication" there:

https://passkeys.dev/docs/reference/terms/#cross-device-authentication-cda

Reply:
Only the CDA text is linked to the term page. FIDO is added for clarity.

Reply:
Is there another kind of CDA in the context of passkeys that requires the clarification? 🤔